### PR TITLE
vet : add check for tabs in text files

### PR DESCRIPTION
**Pull request by eshitachandwani**
- Added check for tabs in text files
- Replaced tabs with spaces in the following file :
   - Documentation/benchmark.md
  
RELEASE NOTES: None

### CODE REVIEW

File: scripts/vet.sh [hunk 1]
```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Check for tabs in Markdown files
+git grep $'\t' -- '*.md' | fail_on_output || echo "No tabs found in Markdown files."
```
Comment (purnesh42H) [inline scripts/vet.sh:74]: shouldn't we be echoing only for failures?

Comment (purnesh42H) [reply scripts/vet.sh:74]: this might be enough `not git grep $'\t' -- '*.md'` because command and comment is self explanatory

Comment (eshitachandwani) [reply scripts/vet.sh]: Done

File: scripts/vet.sh [hunk 2]
```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Check for tabs in Markdown files
+not git grep $'\t' -- '*.md' | fail_on_output
```
Comment (dfawley) [inline scripts/vet.sh:74]: Why pipe to `fail_on_output`?  Inverting the result of `grep` should be sufficient?  I.e. `grep` exits with 0 if there is a match and 1 if there is not.

Comment (eshitachandwani) [reply scripts/vet.sh]: Done

**Review by purnesh42H — APPROVED** [review]
lgtm

File: scripts/vet.sh [hunk 3]
```diff
@@ -70,6 +70,9 @@ not git grep "\(import \|^\s*\)\"google.golang.org/grpc/interop/grpc_testing" --
 # - Ensure that no trailing spaces are found.
 not git grep '[[:blank:]]$'
 
+# - Ensure that no tabs are found in markdown files.
+not git grep $'\t' -- '*.md'
```
Comment (easwars) [inline scripts/vet.sh:74]: Can/Should we do this for all files?

Comment (purnesh42H) [reply scripts/vet.sh:74]: do you mean prohibit tab code files? I think the IDE might be doing that already.

Comment (easwars) [reply scripts/vet.sh:74]: I could be using an editor that doesn't do that. But looks like `gofmt` uses tabs for indentation. See: https://pkg.go.dev/cmd/gofmt. And we run `gofmt` in our vet script. So, we should be good.

**Review by easwars — APPROVED** [review]

File: scripts/vet.sh (hunk 3 above)
Comment (dfawley) [reply scripts/vet.sh:74]: Go pretty famously uses tabs instead of spaces.  Limiting to *.md SGTM.

**Review by dfawley — APPROVED** [review]

**Pull request merged**